package.json

```diff
@@ -14,23 +14,24 @@
     "last 10 Firefox versions"
   ],
   "dependencies": {
+    "@material-ui/lab": "^4.0.0-alpha.57",
     "express": "~4.17.1",
     "socketio": "^1.0.0"
   },
   "devDependencies": {
-    "@material-ui/core": "^4.8.0",
+    "@material-ui/core": "^4.9.10",
     "@material-ui/icons": "^4.5.1",
     "@reduxjs/toolkit": "^1.1.0",
-    "@types/react": "^16.9.17",
+    "@types/react": "^16.8.6 || ^17.0.0",
```

Comment:
Are these actually proper version constraints? If you need `^17.0.0` that is fine, but accepting `^16.8.6` and `^17.0.0` makes no sense then?

Reply:
Changed